resources/fetchersManager/factory/factory.go

```diff
@@ -76,15 +76,26 @@ func getAwsConfig(
 		return awssdk.Config{}, nil, nil
 	}
 
-	awsConfig, err := awsCfgProvider.InitializeAWSConfig(ctx, cfg.CloudConfig.AwsCred)
-	if err != nil {
-		return awssdk.Config{}, nil, fmt.Errorf("failed to initialize AWS credentials: %w", err)
+	var awsConfig awssdk.Config
+	var err error
+	if cfg.Benchmark == config.CIS_AWS {
+		// initialize AWS config without setting the region to be the ec2 metadata region
+		awsConfig, err = aws.InitializeAWSConfig(cfg.CloudConfig.AwsCred)
+		if err != nil {
+			return awssdk.Config{}, nil, fmt.Errorf("failed to initialize AWS credentials: %w", err)
+		}
+	} else {
+		initializedCfg, err := awsCfgProvider.InitializeAWSConfig(ctx, cfg.CloudConfig.AwsCred)
```

Comment:
The `initializedCfg` variable is not really needed, right? You can directly assign to `awsConfig`

Reply:
Our own implementation to `InitializeAWSConfig` is returning a pointer to `aws.Config` while the libbeat implementation is returning a struct, this is the reason for reassigning the var.

Reply:
You could change the var to `var awsConfig *awssdk.Config` but I think it's also fine like that :+1:

Reply:
And then the assignment above will be problematic as we expect a struct, so unfortunately, I think we should keep it the way it is.